src/tests/ftest/server/metadata.py

```diff
@@ -358,19 +351,20 @@ def test_metadata_server_restart(self):
         self.create_pool()
         files_per_thread = 400
         total_ior_threads = 5
-        self.out_queue = queue.Queue()
 
         processes = self.params.get("slots", "/run/ior/clientslots/*")
 
         list_of_uuid_lists = [
             [str(uuid.uuid4()) for _ in range(files_per_thread)]
             for _ in range(total_ior_threads)]
 
+        # Setup the thread manager
+        thread_manager = ThreadManager(run_ior_loop, self.timeout - 30)
+
         # Launch threads to run IOR to write data, restart the agents and
         # servers, and then run IOR to read the data
         for operation in ("write", "read"):
             # Create the IOR threads
-            threads = []
             for index in range(total_ior_threads):
                 # Define the arguments for the ior_runner_thread method
```

Comment:
suggest change ior_runner_thread -> run_ior_loop() in this comment

Reply:
Done